xbmc/epg/EpgInfoTag.cpp

```diff
@@ -50,16 +49,12 @@ CEpgInfoTag::CEpgInfoTag(void) :
     m_iUniqueBroadcastID(-1),
     m_epg(NULL)
 {
-  CPVRChannelPtr emptyChannel;
-  m_pvrChannel = emptyChannel;
-
   CPVRTimerInfoTagPtr emptyTimer;
   m_timer = emptyTimer;
 }
 
 CEpgInfoTag::CEpgInfoTag(CEpg *epg, PVR::CPVRChannelPtr pvrChannel, const CStdString &strTableName /* = StringUtils::EmptyString */, const CStdString &strIconPath /* = StringUtils::EmptyString */) :
     m_bNotify(false),
```

Comment:
would be a good idea to replace PVR::CPVRChannelPtr by const PVR::CPVRChannelPtr & here as another cleanup.